src/components/Data/Documents/ListActions.vue

```diff
@@ -12,14 +12,12 @@
 
       <button class="btn btn-small waves-effect waves-light margin-right-5 primary"
         @click.prevent="$emit('create')"
-        :class="!displayCreate ? 'disabled' : ''"
-        :disabled="!displayCreate"
-        :title="displayCreate ? '' : 'You are not allowed to create a document in this collection'">
+      >
```

Comment:
I don't understand this.

Reply:
I simply copy/paste the existent code. The create button can be disabled when the user does not have the correct rights on the collection.

Reply:
This component was rendered here: https://github.com/kuzzleio/kuzzle-admin-console/pull/444/files/857e6bccb21e13d08dbe2a34ea344c408d5058a6#diff-0805784b9e3841d9b40f8950321b4e38L63

Reply:
It seems to be an error, I corrected that